Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -43,3 +43,23 @@
 </div>
 
 
+@if (Model.MaxPages > 1)
```

Comment:
If you want to you could extract this to a partial. This file isn't too big so not absolutely necessary, but this is the kind of thing you would extract to a partial 🙂